### ISSUE TITLE
Add accessibility linter

### ISSUE THREAD

**Issue by arkid15r** [OWASP/Nest#431]
**Describe the solution you'd like**
Add a workflow to check for accessibility violations in the UI code.

Related to https://github.com/OWASP/Nest/issues/298

**Comment by nitinawari** [comment]
@arkid15r  can i take care of this both this task if @TenzDelek not working on this

**Comment by bhagyashree980** [comment]
Hey @arkid15r,Can you assign this isssue to me i am good in frontend

**Comment by priyankaChauhan2804** [comment]
could i work on this?

**Comment by priyankaChauhan2804** [comment]
hello, I'm not able to solve this issue due to some reasons. If other's want they can work on this issue. Thank you

**Comment by yashgoyal0110** [comment]
Can i look into it?

**Comment by priyankaChauhan2804** [comment]
Hello, i am still looking into this, will update tomorrow about the issue. Sorry for the inconvenience.

**Comment by yashgoyal0110** [comment]
> Hello, i am still looking into this, will update tomorrow about the issue. Sorry for the inconvenience.

you should be sure before before leaving the issue

**Comment by priyankaChauhan2804** [comment]
> > Hello, i am still looking into this, will update tomorrow about the issue. Sorry for the inconvenience.
> 
> you should be sure before before leaving the issue

i still am leaving the issue, i just want to give it one last try till tomorrow.

**Comment by priyankaChauhan2804** [comment]
@yashgoyal0110 you can take it up

**Comment by yashgoyal0110** [comment]
Hey @arkid15r,
I have raised PR for this

**Comment by yashgoyal0110** [comment]
@arkid15r I think u didn't assign this to me?
